examples/usgs-3dep-13/n40w110/n41w106-13/n41w106-13.json

```diff
@@ -0,0 +1,118 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "n41w106-13",
+  "properties": {
+    "start_datetime": "1948-01-01T00:00:00Z",
+    "end_datetime": "2013-12-31T00:00:00Z",
+    "gsd": 10,
+    "proj:epsg": 5498,
+    "proj:shape": [
+      10812,
+      10812
+    ],
+    "proj:transform": [
```

Comment:
I had 9-element transforms in Sentinel-2 on Earth-search as well, but we should be in the habit of dropping the last 3 elements for 2d rasters